packages/eas-cli/package.json

```diff
@@ -131,15 +131,27 @@
     ],
     "helpClass": "./build/help",
     "topics": {
+      "account": {
+        "description": "manage your account"
+      },
       "build": {
         "description": "build app binaries"
       },
       "channel": {
         "description": "manage update channels"
       },
+      "device": {
+        "description": "manage your apple devices"
+      },
+      "project": {
+        "description": "manage your project"
+      },
       "release": {
         "description": "manage update releases"
       },
+      "secret": {
+        "description": "manage project and account secrets"
+      },
```

Comment:
another option is to just remove the descriptions of these topics as they are pretty redundant